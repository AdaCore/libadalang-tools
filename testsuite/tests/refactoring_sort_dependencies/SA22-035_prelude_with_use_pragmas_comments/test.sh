#!/bin/sh

sort_dependencies -P default.gpr -S main_package.ads
