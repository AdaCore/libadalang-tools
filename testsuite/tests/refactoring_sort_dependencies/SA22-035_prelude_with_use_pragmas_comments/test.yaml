title: SA22-035_prelude_with_use_pragmas_comments

description: sort_dependencies tool test

driver: shell_script
