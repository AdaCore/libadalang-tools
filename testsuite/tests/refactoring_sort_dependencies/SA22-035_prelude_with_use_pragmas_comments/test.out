main_package.ads
3:1-52:1 --  Comment 1
with Ada.Containers; --  Comment 2
--  Comment 3
with Ada.Containers.Hashed_Sets;
with Ada.Containers.Vectors;
with Ada.Strings;
with Ada.Strings.Hash;
--  Comment 29
--  Comment 30
with Ada.Strings.Unbounded; --  Comment 6

with Bar; use Bar; --  Comment 24
--  Comment 25
pragma Warnings (Off, --  Comment 26
                 Bar);

--  Comment 16
--  Comment 17
--  Comment 18
with Foo; --  Comment 19
use Foo;
--  Comment 27
--  Comment 28
pragma Elaborate_All (Foo);
pragma Warnings (Off, --  Comment 20
                 Foo);     -- Comment 21
-- Comment 22

private with Ada.Finalization; --  Comment 4
--  Comment 5
private with Ada.Unchecked_Deallocation; --  Comment 7
--  Comment 8

--  Comment 9
--  Comment 10
private with Qux; --  Comment 11
pragma Elaborate (Qux); -- Comment 12

--  Comment 13
--  Comment 14

--  Comment 15

limited with Corge;
--  Comment 23



